packages/lesswrong/lib/collections/comments/permissions.js

```diff
@@ -35,7 +35,8 @@ Users.groups.admins.can(adminActions);
 const sunshineRegimentActions = [
   'comments.softRemove.all',
   'comments.replyOnBlocked.all',
-  'comments.edit.all'
+  'comments.edit.all',
+  'comments.copyConvertedComments.all'
```

Comment:
Misnamed; this is about moving comments, not copying them.

Reply:
fixed